CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contribution
+
+# How to contribute?
+
+## Steps:
+  
+
+  -  Go to [issues](https://github.com/s-ayush2903/Recipebook/issues) & select one upon which you want to work!
+  
+  -  Fork the [repository](https://github.com/s-ayush2903/Recipebook.git)
+  
+  -  Clone the fork [repo](https://github.com/<your_username>/Recipebook)
+     - git clone https://https://github.com/<your_username>/Recipebook
+    
+  -  Create new branch
+     - git checkout -b <Your-Branch-Name>
+  
+  -  Add code/scripts related to your respective issues.
+     - git add <your-contribution>
+ 
+  -  Add a commit message !
+     - git commit -a -m "<Add a message related to changes you made>"
+
+  -  Push changes
+    - git push origin
```

Comment:
Change it to `git push <remote_name> <branch_name>`, this is more specific and precise info for git to tell where to push.

Reply:
Will do.